apps/integration_tests/selenium_cases.py

```diff
@@ -513,6 +520,34 @@ class Action(Enum):
     BROWSERBACK,
 ]
 
+SEQ_CHECK_SCOPES = [
+    {
+        "display": "Check that v2 SMART APP scopes were returned",
+        "action": Action.CONTAIN_TEXT,
+        "params": [20, By.XPATH, X_PATH_FOR_USER_SCOPES, COVERAGE_SCOPE]
+    },
+    {
+        "display": "Check that v2 SMART APP scopes were returned",
+        "action": Action.CONTAIN_TEXT,
+        "params": [20, By.XPATH, X_PATH_FOR_USER_SCOPES, PATIENT_SCOPE]
+    },
+    {
+        "display": "Check that v2 SMART APP scopes were returned",
+        "action": Action.CONTAIN_TEXT,
+        "params": [20, By.XPATH, X_PATH_FOR_USER_SCOPES, EOB_SCOPE]
+    },
+    {
+        "display": "Check that v2 SMART APP scopes were returned",
+        "action": Action.CONTAIN_TEXT,
+        "params": [20, By.XPATH, X_PATH_FOR_USER_SCOPES, OPENID_SCOPE]
+    },
+    {
+        "display": "Check that v2 SMART APP scopes were returned",
+        "action": Action.CONTAIN_TEXT,
+        "params": [20, By.XPATH, "//*[@id='main-content']/div/div/div/pre", "launch/patient"]

```

Comment:
You might have just been messing with this when debugging failing tests, but I think we can go back to using the constant for the path:
```suggestion
        "params": [20, By.XPATH, X_PATH_FOR_USER_SCOPES, "launch/patient"]

```